### PR TITLE
Standardize pro/enterprise labels across docs

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Standardize the pro/enterprise labels across all the relevant docs. Added missing labels for some docs on pro/enterprise features.

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [x] Hotfix

## Notion ticket link

> [Pro/Enterprise labels standardization](https://www.notion.so/rudderstacks/Pro-Enterprise-labels-standardization-d99a2d94e0dc49b3870c4ff5d8bc6524)

### CODE REVIEW

File: docs/data-governance/ruddertyper.mdx [hunk 1]
```diff
@@ -7,7 +7,13 @@ description: >-
 
 # RudderTyper
 
-[RudderTyper](https://github.com/rudderlabs/rudder-typer) is a tool that lets you generate strongly-typed analytics library wrappers based on your [Tracking Plans](https://www.rudderstack.com/docs/data-governance/tracking-plans/). Simply put, it uses an event from your specified tracking plan and generates an analytics call in the supported languages.
+[RudderTyper](https://github.com/rudderlabs/rudder-typer) is a tool that lets you generate strongly-typed analytics library wrappers based on your [Tracking Plans](https://www.rudderstack.com/docs/data-governance/tracking-plans/). 
+
+Simply put, it uses an event from your specified tracking plan and generates an analytics call in the supported languages.
+
+<a href="https://rudderstack.com/enterprise-quote" target="_blank">
```
Comment (ishachopra0401) [inline docs/data-governance/ruddertyper.mdx:14]: As the badge says its available for pro/enterprise users, should we redirect them to:
https://www.rudderstack.com/pricing/

File: docs/data-governance/index.mdx [hunk 2]
```diff
@@ -16,6 +16,10 @@ In reality, however, as multiple stakeholders define and implement the event spe
 - **Capitalization/Casing errors** - For example, one event sets the product name in the lower case, while another sets it in the upper case.
 - **Unit errors** - For example, one event tracks the revenue in pounds, while another tracks it in dollars.
 
+<a href="https://rudderstack.com/pricing" target="_blank">
```
Comment (ishachopra0401) [inline docs/data-governance/index.mdx:19]: Should we redirect users to enterprise quote directly, as done in other enterprise features:
https://rudderstack.com/enterprise-quote

Comment (ameypv-rudder) [inline docs/data-governance/index.mdx:19]:
```suggestion
<a href="https://rudderstack.com/enterprise-quote" target="_blank">
```

File: docs/data-governance/ruddertyper.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/data-governance/ruddertyper.mdx:14]:
```suggestion
<a href="https://www.rudderstack.com/pricing/" target="_blank">
```

File: docs/data-governance/index.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/data-governance/index.mdx]: Agreed- made the change.

**Pull request merged**